### DISCUSSION TITLE
Changing the default Elastic Search log node for storage

### DISCUSSION THREAD

**Discussion by linsecnet** [Security-Onion-Solutions/securityonion#5567] [Unsupported Versions] [answered]
I am new to security onion and I deployed the following in our company: Manager Node, 3-Forward Nodes, 1- Fleet Node, 1- Heavy node, and 1- search node. 

Is there an easy way to change the default elastic search index ( or log location )?

We have 3 available search nodes ( Master, Search, and Heavy Node ) at the moment the search node is the default. But I would like to make the Heavy Node the default because it contains more storage. 

Within Kibana -- Stack Management -- Remote Clusters - I see all three there.

Logs are all coming in with no issues, but the index of the logs is all showing the search node.

The plan is to use the search node as cold storage for aged logs ( that is the plan at least )

Any guidance will be highly appreciated.

**Comment by dougburks** [comment, marked as answer]
A `manager` node is not intended to store any real data.

A `heavy` node is only intended to store data that it generates locally or that it received directly. It's not intended to pull logs from the redis queue on the manager the way that a `search` node is.

For more information, please see https://docs.securityonion.net/en/2.3/architecture.html.
